src/charm.py

```diff
@@ -72,22 +72,7 @@ def __init__(self, *args):
                 ServicePort(name="prometheus-exporter", port=PROMETHEUS_PORT),
             ],
         )
-        self._kubernetes_multus = KubernetesMultusCharmLib(
-            charm=self,
-            container_name=self._bessd_container_name,
-            cap_net_admin=True,
-            network_annotations=[
-                NetworkAnnotation(
-                    name=ACCESS_NETWORK_ATTACHMENT_DEFINITION_NAME,
-                    interface=ACCESS_INTERFACE_NAME,
-                ),
-                NetworkAnnotation(
-                    name=CORE_NETWORK_ATTACHMENT_DEFINITION_NAME,
-                    interface=CORE_INTERFACE_NAME,
-                ),
-            ],
-            network_attachment_definitions_func=self._network_attachment_definitions_from_config,
-        )
+        self._kubernetes_multus = self._get_kubernetes_multus()
```

Comment:
Probably an individual thing, but I liked the previous version more :) It would give an instant view on the NADs created by this charm.

Reply:
I got back that change.